packages/mdl-rtl/README.md

```diff
@@ -0,0 +1,218 @@
+# MDL RTL
+
+MDL RTL provides sass mixins to assist with RTL / bi-directional layouts within MDL components.
+Because we would like to achieve a standard approach to RTL throughout MDL, we highly recommend
+that any MDL component that needs RTL support leverage this package.
+
+## Installation
+
+> Installation via npm will be available post-alpha
+
+## Usage
+
+Simply `@import "mdl-rtl/mixins";` and start using the mixins. Each mixin is described below.
+
+### mdl-rtl
+
+```scss
+@mixin mdl-rtl($root-selector: null)
+```
+
+Creates a rule that will be applied when an MDL component is within the context of an RTL layout.
+
+Usage Example:
+
+```scss
+.mdl-foo {
+  position: absolute;
+  left: 0;
+
+  @include mdl-rtl {
+    left: auto;
+    right: 0;
+  }
+
+  &__bar {
+    margin-left: 4px;
+    @include mdl-rtl(".mdl-foo") {
+      margin-left: auto;
+      margin-right: 4px;
+    }
+  }
+}
+
+.mdl-foo--mod {
+  padding-left: 4px;
+
+  @include mdl-rtl {
+    padding-left: auto;
+    padding-right: 4px;
+  }
+}
+```
+
+will emit the following css:
+
+```css
+.mdl-foo {
+  position: absolute;
+  left: 0;
+}
+[dir="rtl"] .mdl-foo, .mdl-foo[dir="rtl"] {
+  left: auto;
+  right: 0;
+}
+.mdl-foo__bar {
+  margin-left: 4px;
+}
+[dir="rtl"] .mdl-foo .mdl-foo__bar, .mdl-foo[dir="rtl"] .mdl-foo__bar {
+  margin-left: auto;
+  margin-right: 4px;
+}
+
+.mdl-foo--mod {
+  padding-left: 4px;
+}
+[dir="rtl"] .mdl-foo--mod, .mdl-foo--mod[dir="rtl"] {
+  padding-left: auto;
+  padding-right: 4px;
+}
+```
+*N.B.**: checking for `[dir="rtl"]` on an ancestor element works in most cases, it will sometimes
+lead to false negatives for more complex layouts, e.g.
+
+```html
+<html dir="rtl">
+  <!-- ... -->
+  <div dir="ltr">
+    <div class="mdl-foo">Styled incorrectly as RTL!</div>
+  </div>
+</html>
+```
+
+Unfortunately, we've found that this is the best we can do for now. In the future, selectors such
+as [:dir](http://mdn.io/:dir) will help us mitigate this.
+
+### mdl-rtl-reflexive-box
```

Comment:
That's actually what it was originally, and then I changed it because of its length 😄